tools/common/Frameworks.cs

```diff
@@ -211,6 +212,7 @@ public static Frameworks GetiOSFrameworks (bool is_simulator_build)
 				{ "AddressBookUI", "AddressBookUI", 3 },
 				{ "AudioToolbox", "AudioToolbox", 3 },
 				{ "AVFoundation", "AVFoundation", 3 },
+				{ "CFNetwork", "CFNetwork", 3 },
```

Comment:
iirc it was unconditionally added because of the StartWWAN code

Reply:
> iirc it was unconditionally added because of the StartWWAN code

Correct:

https://github.com/xamarin/xamarin-macios/blob/bf34d86323f532eab95a73e8f35885a8f1bc641d/tools/common/CompilerFlags.cs#L161-L162

It's still valuable to have a complete list of frameworks here though (less confusing when you start wondering why a framework isn't listed).

Reply:
yeah, unless the dupe cause an issue in our (or later Apple) logic - which I don't think is a problem.

Alternative would be just to add it as a comment.